src/subcommands/util.rs

```diff
@@ -126,12 +126,23 @@ impl<'a> UtilSubCommand<'a> {
                     .arg(
                         binary_hex_arg
                             .clone()
+                            .required(false)
+                            .required_unless("utf8-string")
+                            .conflicts_with("utf8-string")
                             .about("The data to be signed (blake2b hashed with 'ckb-default-hash' personalization)")
                     )
                     .arg(
                         Arg::with_name("no-magic-bytes")
                             .long("no-magic-bytes")
                             .about("Don't add magic bytes before binary data (magic bytes: \"Nervos Message:\")")
+                    )
+                    .arg(
+                        Arg::with_name("utf8-string")
+                         .long("utf8-string")
+                         .takes_value(true)
+                         .required_unless(binary_hex_arg.get_name())
+                         .conflicts_with(binary_hex_arg.get_name())
+                         .about("The utf-8 string to be signed (blake2b hashed with 'ckb-default-hash' personalization)")
```

Comment:
The help message is a bit confusing whether the user is required to hash the string first.

```suggestion
                         .about("The utf-8 string to be signed. The string will be hashed using blake2b with 'ckb-default-hash' personalization first.")
```